vagrant/create_kubeconfig.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Create ~/.kube/config file that uses the https endpoint to make
+# sure all requests go through rbac validation (requests via the
+# http endpoint bypass all validation).
+
+set -o errexit -o nounset -o xtrace
+
+SECRET=$(kubectl get sa default -n kube-system -o jsonpath="{.secrets[0].name}")
+CA_CRT=$(kubectl get secrets ${SECRET} -n kube-system -o jsonpath="{.data['ca\.crt']}")
+TOKEN=$(kubectl get secrets ${SECRET} -n kube-system -o jsonpath="{.data['token']}" | base64 -d)
```

Comment:
Nit: use full names for scripts
```suggestion
SECRET=$(kubectl get serviceaccount default --namespace kube-system --output jsonpath="{.secrets[0].name}")
CA_CRT=$(kubectl get secrets ${SECRET} --namespace kube-system --output jsonpath="{.data['ca\.crt']}")
TOKEN=$(kubectl get secrets ${SECRET} --namespace kube-system --output jsonpath="{.data['token']}" | base64 -d)
```